src/zero/api/routes.clj

```diff
@@ -121,10 +122,14 @@
             :swagger {:tags ["Information"]}}}]
    ["/version"
     {:get  {:summary "Get the versions of server and supported pipelines"
-            :handler (handlers/success (zero/get-the-version))
-            :responses {200 {:body {:version string?
-                                    :build    pos-int?
-                                    :built    string?}}}
+            :handler (handlers/success (let [versions (zero/get-the-version)
+                                             pipeline-versions-keys (keep (fn [x] (when (and (string? x) 
+                                                                                             (str/ends-with? x ".wdl")) x)) 
+                                                                          (keys versions))]
+                                         {:pipeline-versions (select-keys versions pipeline-versions-keys)
+                                          :version (apply dissoc versions pipeline-versions-keys)}))
+            :responses {200 {:body {:version map?
+                                    :pipeline-versions map?}}}
```

Comment:
Seems like the wrong place to separate these.
Either do it when constructing the version itself,
or do it in the UI, which is the code that cares.
Also, map? is too loose.
We want /version to at least return a :version.

Reply:
> map? is too loose.

That makes sense, will fix it in a follow up PR.

> Either do it when constructing the version itself,
or do it in the UI, which is the code that cares.

The problem I had with doing this in the frontend is that we have to use specific getters like: https://github.com/broadinstitute/wfl/pull/28/files#diff-88ce83892fa39769e0067bd6efa72316L35 which will require updates every time we add or update a pipeline (name). If that's fine with you I could revert it!